deploy/central/tasks-chart/templates/task-ods-build-typescript-with-sidecar.yaml

```diff
@@ -72,6 +72,15 @@ spec:
     description: 'Node.js version to use - supported versions: 16'
     name: node-version
     type: string
+  - default: dist
+    description: The directory into which `npm run build` places files. The files
+      inside `build-dir` will be copied to `output-dir`.
+    name: build-dir
+    type: string
+  - default: "false"
+    description: Whether `node-modules` is copied or not.
+    name: copy-node-modules
+    type: string
```

Comment:
I think it would be great if the task description contains some information about these parameters. Along the lines of "For frontend components, do ...., for backend components, do ...". What do you think?